pkg/odp/segment_manager.go

```diff
@@ -19,10 +19,11 @@ package odp
 
 import (
 	"fmt"
-
-	"github.com/optimizely/go-sdk/pkg/odp/utils"
 )
 
+// SMOptionConfig are the SegmentManager options that give you the ability to add one more more options before the segment manager is initialized.
+type SMOptionConfig func(em *DefaultSegmentManager)
```

Comment:
shouldn't this be SMConfigOptions

Reply:
Changing it into `SMOptionFunc` which is in sync with the older options in the sdk.